packages/styles/src/variables/components/_alert.scss

```diff
@@ -1,51 +1,37 @@
-@use './close';
-@use './../commons';
 @use './../color';
+@use './../commons';
 @use './../spacing';
 @use './../type';
 
-// Alerts and Toasts
-//
-// Define alert colors, border radius, and padding.
+$alert-padding-x: spacing.$size-small-large !default;
+$alert-padding-y: spacing.$size-regular !default;
+$alert-margin-bottom: spacing.$spacer !default;
+$alert-box-shadow: 0 2px 4px 0 rgba(color.$black, 0.2) !default;
+$alert-border-radius: commons.$border-radius !default;
+$alert-gap-x: spacing.$size-small-large !default;
+$alert-gap-y: spacing.$size-micro !default;
+$alert-font-size: type.$font-size-base !default;
+$alert-font-weight: type.$font-weight-light !default;
+
+$alert-heading-font-size: type.$font-size-bigger-regular !default;
+$alert-heading-font-weight: type.$headings-font-weight !default;
 
-$alert-padding-y: spacing.$size-small-large !default; // Design System
-$alert-padding-x: spacing.$size-large !default; // Design System
-$alert-toast-padding-y: spacing.$size-small-regular !default; // Design System only
-$alert-toast-padding-x: spacing.$size-regular !default; // Design System only
+$alert-link-font-weight: type.$font-weight-normal !default;
 
-$alert-margin-bottom: 1rem !default; // Design System
-$alert-margin: $alert-margin-bottom; // Design System only
-$alert-border-radius: commons.$border-radius !default;
-$alert-link-font-weight: type.$font-weight-bold !default;
-$alert-border-width: commons.$border-width !default;
+$alert-hr-margin-top: spacing.$size-mini !default;
+$alert-hr-margin-bottom: spacing.$size-small-regular !default;
+
+$alert-icon-size: spacing.$size-small-huge !default;
 
-$alert-bg-level: -10 !default;
-$alert-border-level: -9 !default;
-$alert-color-level: 6 !default;
+$alert-action-button-margin-start: spacing.$size-mini !default;
 
-$alert-list: (
+$alert-variants: (
   'primary' color.$gray-80 3134,
   'notification' color.$gray-80 3134,
   'success' color.$success 2105,
   'danger' color.$error 2104,
   'error' color.$error 2104,
   'warning' color.$warning 2104,
-  'info' color.$info 2106
-); // Design System only
-
-$alert-line-height: type.$line-height-copy; // Design System only
-$alert-text-space-between: spacing.$size-micro;
-
-$alert-icon-size: spacing.$size-small-huge; // Design System only
-$alert-close-icon-size: close.$btn-close-width + spacing.$size-micro; // Design System only
-
-$alert-toast-icon-size: spacing.$size-big; // Design System only
-
-$alert-font-size: type.$font-size-bigger-regular; // Design System only
-$alert-font-size-small: type.$font-size-small; // Design System only
-
-$alert-toast-width: 25rem; // Design System only
-$alert-toast-width-small-breakpoint: 21rem; // Design System only
-
-$alert-toast-box-shadow-hover: 0 0 1rem 0 rgba(var(--post-contrast-color-rgb), 0.4); // Design System only
-$alert-close-hover-opacity: 0.4; // Design System only
+  'info' color.$info 2106,
+  'cookies' color.$gray-background 2201
```

Comment:
I would like to have a more generic name for this new color variable.
Something like `gray` or `light`...

Because it can be used for other topics in the future and then it will feel wrong to use the given `alert-cookies` class.